integrationTests/consensus/testInitializer.go

```diff
@@ -168,7 +169,14 @@ func createAccountsDB(marshalizer marshal.Marshalizer) state.AccountsAdapter {
 		SnapshotsBufferLen: 10,
 		MaxSnapshots:       2,
 	}
-	trieStorage, _ := trie.NewTrieStorageManager(store, marshalizer, hasher, cfg, generalCfg)
+	trieStorage, _ := trie.NewTrieStorageManager(
+		store,
```

Comment:
if we are at refactor - this could have got an Argument structure as input instead of  multi arg

Reply:
Done.